### PR TITLE
Thleen2

### PR DESCRIPTION
**Pull request by tetsuroescudero**
backend not working

### CODE REVIEW

File: client/src/components/Employees/Employees.js [hunk 1]
```diff
@@ -1,13 +1,90 @@
...
+      height: "45%",
+      width: "32%",
+      transform: "translate(-50%, -50%)"
+    }
+  };
+
+  return (
+    <div>
+      <div className="addbutton">
+        <button onClick={openModal}>+ Add Employees</button>
```
Comment (bdeleonardis1) [inline client/src/components/Employees/Employees.js]: This button doesn't look styled at all for me. Can we make it look good?

File: client/src/components/Employees/Employees.js [hunk 2]
```diff
@@ -1,13 +1,90 @@
...
+      </div>
+      <Modal
```
Comment (bdeleonardis1) [inline client/src/components/Employees/Employees.js]: The modal has a horizontal and vertical scroll bar for me. Let's fix that. If you can't see it on your computer, try resizing the window until you do.

File: client/src/components/Employees/Employees.js [hunk 3]
```diff
@@ -1,13 +1,90 @@
...
+            <div className="MoffittButton">
+              <input type="radio" id="moffitt" name="moffitt" value="moffitt" />
+            </div>
+            <h3 className="DoeText">Doe</h3>
+            <div className="DoeButton">
+              <input type="radio" id="doe" name="doe" value="doe" />
+            </div>
+          </div>
+        </div>
+      </Modal>
```
Comment (bdeleonardis1) [inline client/src/components/Employees/Employees.js]: We also need a button for them to submit the form and a cancel button for them to get out of the form.

File: client/src/components/Employees/Employees.js (hunk 1 above)
Comment (tetsuroescudero) [reply client/src/components/Employees/Employees.js]: done

File: client/src/components/Employees/Employees.js (hunk 2 above)
Comment (tetsuroescudero) [reply client/src/components/Employees/Employees.js]: done

File: client/src/components/Employees/Employees.js (hunk 3 above)
Comment (tetsuroescudero) [reply client/src/components/Employees/Employees.js]: done

**Comment by tetsuroescudero** [comment]
RFAL!

**Pull request merged**